src/utils/resolveYextEntityField.ts

```diff
@@ -0,0 +1,37 @@
+import { EntityFieldType } from "../components/YextEntityFieldSelector.tsx";
+
+export const resolveYextEntityField = <T>(
+  document: any,
+  entityField: EntityFieldType
+): T => {
+  // return static value if "Use Static Value" selected
+  if (entityField.fieldName === "") {
```

Comment:
Does the user have to unset the fieldName to use the static value?

Is this what we want?

Reply:
If both staticValue and fieldName are set, the fieldName will be chosen.

Is this what we want?

Reply:
Yes. And if it's not in the document it still falls back to staticValue.